.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       POSTGRES_HOST: localhost
       POSTGRES_PORT: 5432
       JWT_SECRET: test_secret_for_CI
+      DATABASE_URL: postgresql://postgres:postgres@localhost:5432/vsc?schema=public
```

Comment:
We should have quotes over the URL

Reply:
Actually it looks like that's not the problem